### PR TITLE
[Features] Upload avatar to profile

### PR DESCRIPTION
**Pull request by B0RlS**
Ticket: 
https://trello.com/c/NRCPMXgw/9-features-upload-photos-to-profile

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 # Ignore bundler config.
 /.bundle
+/.idea
```
Comment (labs-ruby) [inline .gitignore]: Move env-specific stuff to global gitignore

File: config/application.rb [hunk 2]
```diff
@@ -14,6 +16,7 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
 
+    config.assets.initialize_on_precompile = false
```
Comment (labs-ruby) [inline config/application.rb]: Please leave a comment that this option is disabled due to Heroku

File: .gitignore (hunk 1 above)
Comment (B0RlS) [reply .gitignore]: done

File: config/application.rb (hunk 2 above)
Comment (B0RlS) [reply config/application.rb]: done

**Comment by B0RlS** [comment]
>     1. Please add specs
> 
>     2. I saw you've added gravatar gem, but I didin't find any gravatar image processing implementation

1. WIP
2. Check views/profiles/show.html.erb , line 6

**Comment by B0RlS** [comment]
[[Features] Upload avatar to profile](https://trello.com/c/NRCPMXgw/9-features-upload-avatar-to-profile)

**Comment by B0RlS** [comment]
[[Features] Upload avatar to profile](https://trello.com/c/NRCPMXgw/9-features-upload-avatar-to-profile)

**Pull request merged**